Dockerfile

```diff
@@ -0,0 +1,53 @@
+ARG	GIT_VERSION=unknown
+ARG	BUILD_TARGET
+ARG	FROM_LOCAL_PARENT
+
+FROM	golang:1-bullseye	as	gobuild
+
+ARG TARGETARCH
+
+# Download c2patool needed to sign our C2PA manifest
+# We download it from any of our previous builds, because building c2patool from source is very slow with QEMU
+RUN	apt update && apt install -yqq \
+	curl \
+	ca-certificates \
+	&& curl https://build.livepeer.live/c2patool/0.6.2/c2patool-linux-${TARGETARCH}.tar.gz -o /c2patool.tar.gz \
+	&& tar xzf /c2patool.tar.gz
+
+WORKDIR	/src
+
+ADD	go.mod go.sum	./
+RUN	go mod download
+
+ADD	.	.
+RUN make build
+
+ARG	GIT_VERSION
+ENV	GIT_VERSION="${GIT_VERSION}"
+
+FROM	ubuntu:22.04	AS	catalyst
```

Comment:
I copied it the base image from `livpeeer/catalyst`, @hjpotter92 @pwilczynskiclearcode do you think we should try a smaller image?

Reply:
Do we want to try some liter (apline?) images? Do you know if it could work? @hjpotter92 Do you maybe know?

Reply:
we stopped using alpine because it made debugging on the pods difficult. devs. needed to install base packages before they could even do top/kill etc.